openssl-sys/build.rs

```diff
@@ -20,10 +20,14 @@ fn main() {
         }
     }
 
-    let (libcrypto, libssl) = if target.contains("windows") {
-    	("eay32", "ssl32")
-    } else {
-    	("crypto", "ssl")
+    let libs_env = env::var("OPENSSL_LIBS").ok();
+    let libs = match libs_env {
+        Some(ref v) => v.split(":").collect(),
+        None => if target.contains("windows") && env::var_os("MSYSTEM").is_none() {
```

Comment:
Is the MSYSTEM check the right thing to do here? I've never been able to successfully link against the Msys64-installed libcrypto and libssl.